monitoring/dr/alerts.test.yaml

```diff
@@ -0,0 +1,87 @@
+# These tests are meant to go hand-in-hand with the rendered alert rule.
+# Use github.com/scality/action-prom-render-test@python-renderer python module
+#
+# Render the alerts file with
+# gen-alert render alerts.yaml
+evaluation_interval: 1m
+rule_files:
+  - alerts.rendered.yaml
+tests:
+  - interval: 1m
+    input_series:
+      - series: up{drSinkInstance="", job="artesca-data-dr-source-base-queue-connector-metrics", namespace="zenko"}
+        values:  _x7
+      - series: up{drSinkInstance="artesca-data-dr", job="artesca-data-dr-base-queue-connector-metrics", namespace="zenko"}
+        values:  1x7
+      - series: kafka_connect_task_error_total_record_errors{drSinkInstance="", job="artesca-data-dr-source-base-queue-connector-metrics"}
+        values:  0 1 2 2 2 2 2
+      - series: kafka_connect_task_error_total_record_failures{drSinkInstance="", job="artesca-data-dr-source-base-queue-connector-metrics"}
+        values:  0x7
+      - series: kafka_connect_task_error_total_record_errors{drSinkInstance="artesca-data-dr", job="artesca-data-dr-base-queue-connector-metrics"}
+        values:  0 1 2 2 2 2 2
+      - series: kafka_connect_task_error_total_record_failures{drSinkInstance="artesca-data-dr", job="artesca-data-dr-base-queue-connector-metrics"}
+        values:  0 1x6
+      - series: s3_lifecycle_last_timestamp_ms{drSinkInstance="", job="artesca-data-backbeat-lifecycle-.*-headless", namespace="zenko"}
+        values:  10000000 10000000 10000000
+      - series: mongodb_ss_repl_lastWrite_lastWriteDate{drSinkInstance="artesca-data-dr", job="zenko/data-db-mongodb-sharded-shard.*", namespace="zenko"}
+        values:  5000000 5000000 5000000
+    alert_rule_test:
+      - alertname: DrResourcePausedWhileOtherRunning
+        eval_time: 1m
+        exp_alerts:
+          - exp_labels:
+              severity: warning
+            exp_annotations:
+              summary: 'DR Resource Paused While Other Running'
+              description: 'One site s DR resource is paused while the other is not. This could lead to data inconsistency between sites.'
+      - alertname: DrResourcePausedForTooLong
+        eval_time: 4m59s
+        exp_alerts: []
+      - alertname: DrResourcePausedForTooLong
+        eval_time: 5m 
+        exp_alerts:
+          - exp_labels:
+              severity: "warning"
+            exp_annotations:
+              summary: 'DR resource paused for too long'
+              description: 'The DR resource has been paused for more than 5 minutes.'
```

Comment:
(Not related to the tests directly)
Is too long === 5min? Or can we check with CNES their oplog history size and avoid having too small durations here? I think, if we need to pause the DR, that the (manual) operations to be done will likely take more than 5min, and if CNES has an oplog history in minutes (and not days/hours), it's an issue...

(Or better if we can use kafka+mongodb metrics to dynamically detect if we are approaching the end of the oplog...)

Reply:
"too long" is hard to tell indeed, and should not be based on CNES.

generally, I think it is fair to report an "indicative" alert that we are indeed paused (i.e. the system is not really "live", even if it is not an issue).
but indeed we should ideally make it critical when "the pause duration gets close to the available mongo oplog capacity" (somewhat easier said than done)